mods/ra2/weapons/zaps.yaml

```diff
@@ -170,13 +170,13 @@ SuperComet:
 
 RadBeamWeapon:
 	ValidTargets: Ground
+	InvalidTargets: Structure, Wall, ImmuneToRadiation
```

Comment:
Units that were immune to radiation were still targetable by Desolator in the original game, so move this to warhead.

Reply:
Also Harvesters and Other Dedolators should be immune to rad too.

Reply:
This will probably cause problems with the AI, but okay.